synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -518,6 +518,27 @@ def recur_animate(self, anim_type):
                 self.expression = ret
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "atan2":
+                self.subparams["atan2"].extract_subparams()
+                y, eff_1 = self.subparams["atan2"].subparams["y"].recur_animate(anim_type)
+                x, eff_2 = self.subparams["atan2"].subparams["x"].recur_animate(anim_type)
+                self.expression_controllers.extend(eff_1)
+                self.expression_controllers.extend(eff_2)
+                
+                ret = "div(Math.atan2({y},{x}),degreesToRadians)"
```

Comment:
This expression does not seem right to me, Hence I tested changing the angle of a group layer, and it does not change anything. I guess you need to use something like:
`ret = "radiansToDegrees(Math.atan2({y}, {x}))"`
You might need to modify the above expression accordingly.